pkgs/development/tools/misc/distcc/default.nix

```diff
@@ -1,21 +1,25 @@
-{ stdenv, fetchurl, popt, avahi, pkgconfig, python, gtk, runCommand, gcc
+{ stdenv, fetchFromGitHub, popt, avahi, pkgconfig, python, gtk, runCommand, gcc, autoconf, automake, which
 , sysconfDir ? ""   # set this parameter to override the default value $out/etc
 , static ? false
 }:
 
 let
   name    = "distcc";
-  version = "3.1";
+  version = "2016-02-24";
   distcc = stdenv.mkDerivation {
     name = "${name}-${version}";
-    src = fetchurl {
-      url = "http://distcc.googlecode.com/files/${name}-${version}.tar.bz2";
-      sha256 = "f55dbafd76bed3ce57e1bbcdab1329227808890d90f4c724fcd2d53f934ddd89";
+    src = fetchFromGitHub {
+      owner = "distcc";
+      repo = "distcc";
+      rev = "b2fa4e21b4029e13e2c33f7b03ca43346f2cecb8";
+      sha256 = "1vj31wcdas8wy52hy6749mlrca9v6ynycdiigx5ay8pnya9z73c6";
     };
 
-    buildInputs = [popt avahi pkgconfig python gtk];
+    buildInputs = [popt avahi pkgconfig python gtk autoconf automake pkgconfig which];
     preConfigure =
     ''
+      export CPATH=${gcc.cc}/lib/gcc/x86_64-unknown-linux-gnu/${gcc.cc.version}/plugin/include
```

Comment:
This doesn't look right. We cannot hard-code the "x86_64-unknown-linux-gnu" bit in this build.